e2e/portalicious/pages/TableComponent.ts

```diff
@@ -0,0 +1,64 @@
+import { Locator } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+
+class TableComponent extends BasePage {
+  readonly page: Page;
+  readonly tableRows: Locator;
+  readonly goToProfileOption: Locator;
```

Comment:
The tableComponent I think should be "Ignorant" of what types of content it has. Ideally, this same table component is used in all of the pages with a table (eg. teams, activity log, etc.), so it shouldn't have any functions/attributes specific to the content.